livemaker/patch.py

```diff
@@ -89,17 +89,17 @@ def bar_show(item):
                 for info in bar:
                     # data = orig_lm.read(info, decompress=False)
                     # new_lm.writebytes(info, data)
-                    if info.path == lsb_path:
+                    if os.path.isfile(info.path):#if file exists in current folder
```

Comment:
info.path is already a [pathlib.Path](https://docs.python.org/3/library/pathlib.html) so the pathlib functions should be used instead of os.path

(something like)
```
if info.path.is_file():
```